src/pytorch_lightning/trainer/connectors/callback_connector.py

```diff
@@ -148,10 +151,10 @@ def _configure_model_summary_callback(self, enable_model_summary: bool) -> None:
         progress_bar_callback = self.trainer.progress_bar_callback
         is_progress_bar_rich = isinstance(progress_bar_callback, RichProgressBar)
 
+        model_summary: ModelSummary = ModelSummary()
```

Comment:
Don't assign here, it's only for purpose of typing
```suggestion
        model_summary: ModelSummary
```